cmake/StandardProjectSettings.cmake

```diff
@@ -32,6 +32,8 @@ if(ENABLE_IPO)
     message(SEND_ERROR "IPO is not supported: ${output}")
   endif()
 endif()
+
+# TODO: do not use global add_compiler_options()! CK
```

Comment:
please note

Reply:
This is a good catch; #57 really should have used something more like `target_compile_options(project_options ...)` for this part instead.

However, I would prefer if you raised this issue in the [Issues](https://github.com/cpp-best-practices/cpp_starter_project/issues) section, rather than adding comments.

This is a very large PR, and fixing this problem is probably out of scope. If you were to fix this problem in a separate PR, I'm sure it would be merged much more quickly than this PR.